README.md

```diff
@@ -19,17 +19,48 @@ $> npm install react-onclickoutside --save
 var onClickOutside = require('react-onclickoutside');
 
 // create a new component, wrapped by this onclickoutside HOC:
-var MyComponent = onClickOutside(React.createClass({
+var MyComponent = onClickOutside(
```

Comment:
That looks a little different in terms of use-case, though - I'd rather we change this such that `onClickOutside` has the signature `onClickOutside(componentClass, [options])`, maintaining backward compatibility.